src/DebugEngineHost/HostMarshal.cs

```diff
@@ -68,5 +68,18 @@ public static IDebugEventCallback2 GetThreadSafeEventCallback(IDebugEventCallbac
         {
             return new VSImpl.VSEventCallbackWrapper(ad7Callback);
         }
+
+        /// <summary>
+        /// Allocate storage and copy the guid to the allocated bytes
+        /// </summary>
+        /// <param name="guid">guid to copy</param>
+        /// <returns>pointer to the allocated bytes</returns>
+        public static IntPtr AllocateGuid(Guid guid)
```

Comment:
Is this still needed? I don't see a call to it anywhere while I was trying to verify it was freed.